app/assets/stylesheets/custom.css.scss

```diff
@@ -88,7 +92,7 @@ a:hover, a:active {
     padding-left: 10px;
 }
 
-#cart:hover {
+#cart:hover, #cart:focus {
```

Comment:
Avoid using id selectors<br>Each selector in a comma sequence should be on its own line